src/status_im2/navigation/view.cljs

```diff
@@ -69,12 +69,17 @@
         (when js/goog.DEBUG
           [reloader/reload-view])]))))
 
+; Designs require bottom inset to be bigger than safe area, otherwise it is too close to the bottom
```

Comment:
Hmm I can understand why you did this, to be honest in most cases i do this too.

The thought process behind my original comment was this ,
Since `safe_area.cljs` is an interface, I thought it would be better to have a bottom sheet variant over there which could then be used like this in your code like this :

```clojure
(def bottom-sheet
  (reagent/reactify-component
   (fn []
     (let [{:keys [sheets hide?]} (rf/sub [:bottom-sheet])
           sheet                  (last sheets)
           insets                 (safe-area/bottom-sheet-insets)]
       ^{:key (str "sheet" @reloader/cnt)}
       [:<>
        [inactive]
```

But totally upto you, I don't have a very strong opinion on this :)